RoboJackets/robocup-software
# unassigned Role is Seeker Role

## Description
This makes the seeker role the default for all robots that are not being used by a tactic. This allows for robots to "get open" so they are viable for the receiver role in a pass tactic. 

## Steps to test
### Test Case 1
1. Run the Offense play and ensure that all robots are not being assigned a tactic are "seeking". 
2. Make sure that the seeker role is NOT being assigned to a robot that is already assigned another role!
3. Run the Defense play and change the number of wallers to see how the seekers behave.

Expected result: All robots have some role to do either seeking or the ones assigned by the play. 

##Sub-issues
- [x] A whole Formation object is being passed into the Seek Tactic, ensure that its implementation makes sense
